### PR TITLE
Features/arm tray

### PR DESCRIPTION
**Pull request by hello-garv**
Added documentation for the arm tray designed for stretch to help shuttle miscellaneous items and assist a care partner.

### CODE REVIEW

File: tool_share/arm_mounted_tray_V1/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<img src="./images/bringing_condiments.jpg" alt="Stretch carrying condiments" height="400" />
+
+This tray conveniently clips onto Stretch's first arm link to provide a surface for carrying objects, including two hooks in the front and a cup holder to assist a variety of tasks. 
+
+## Parts List
+
+| Item | Qty | Vendor           |
+| ------------- |:-------------:| -----: |
+| [Tray](CAD/stretch_tray.STL) | 1 | PLA 3D Printer |
+| [Arm Clips](CAD/tray_clip.STL) | 2 | PLA 3D Printer |
```
Comment (hello-binit) [inline tool_share/arm_mounted_tray_V1/README.md:18]: Looks like these STLs are missing from the CAD folder.

**Review by hello-binit — CHANGES_REQUESTED** [review]
@hello-garv Cool! I left some feedback here.

File: tool_share/arm_mounted_tray_V1/README.md (hunk 1 above)
Comment (hello-garv) [reply tool_share/arm_mounted_tray_V1/README.md:18]: Oops, thanks for catching this! I updated the files in the CAD folder.

**Review by hello-binit — APPROVED** [review]
Nice! Looks good to merge!

**Pull request merged**